src/Uno.UWP/UI/Input/GestureRecognizer.Manipulation.cs

```diff
@@ -202,19 +255,40 @@ private ManipulationDelta GetDelta(Points from)
 				};
 			}
 
-			private bool IsSignificant(ManipulationDelta delta)
-				=> Math.Abs(delta.Translation.X) >= MinManipulationDeltaTranslateX
-				|| Math.Abs(delta.Translation.Y) >= MinManipulationDeltaTranslateY
-				|| delta.Rotation >= MinManipulationDeltaRotate // We used the ToDegreeNormalized, no need to check for negative angles
-				|| Math.Abs(delta.Expansion) >= MinManipulationDeltaExpansion;
+			private ManipulationDelta GetDelta(ManipulationDelta cumulative)
+			{
+				var deltaSum = _sumOfPublishedDelta;
+
+				var translateX = _isTranslateXEnabled ? cumulative.Translation.X - deltaSum.Translation.X : 0;
+				var translateY = _isTranslateYEnabled ? cumulative.Translation.Y - deltaSum.Translation.Y : 0;
+				var rotate = _isRotateEnabled ? cumulative.Rotation - deltaSum.Rotation : 0;
+				var scale = _isScaleEnabled ? cumulative.Scale / deltaSum.Scale : 1;
+				var expansion = _isScaleEnabled ? cumulative.Expansion - deltaSum.Expansion : 0;
+
+				return new ManipulationDelta
+				{
+					Translation = new Point(translateX, translateY),
+					Rotation = (float)MathEx.NormalizeDegree(rotate),
+					Scale = scale,
+					Expansion = expansion
+				};
+			}
+
+			internal struct Thresholds
```

Comment:
I think this should be singular, Threshold is a struct representing one threshold, if I understand correctly

Reply:
Oh, seeing it's usage later I realize its 4 different thresholds? So ignore my previous comment.

Do we need TranslateX and Y or they are just a Translate Threshold?

Reply:
We have to check them both anyway, so even if I agree that usually they are the same, lets customize them. Anyway it's only private constants (actually `static readonly`) stuff, so it does not add any complexity to user.